conda/recipes/cudf/meta.yaml

```
@@ -82,7 +82,7 @@ requirements:
     - typing_extensions >=4.0.0
     - pandas >=2.0,<2.2.3dev0
     - cupy >=12.0.0
-    - numba >=0.57
+    - numba-cuda >= 0.0.13
```

extra space here is tripping it up, I think

```suggestion
    - numba-cuda >=0.0.13
```